linera-storage-service/src/child.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use linera_service::util::CommandExt;
+use std::time::Duration;
+use tokio::process::{Child, Command};
+
+/// Configuration for a storage service running as a child process
+pub struct StorageServiceChild {
```

Comment:
The expected name for this would be `StorageServiceConfig`.

(Also, it's not completely clear to me that this config object is useful, but ok.)

Reply:
I choose to name it `StorageServiceSpanner`.